IA-1804 convert InstancePopoverComponent to typescript

olethanh opened the pull request:
It is displayed on top of file in the submission page - > file tab -> open an image preview -> Popup button on the bottom right

![image](https://user-images.githubusercontent.com/82500/207872230-079a53f2-e5b6-4a11-a07c-9f4797697890.png)

kaiobrito commented:
It does solve the front-end issue but I get a `ValueError: Bad GPKG group ['PML95xY3pNy', 'hraO6yWAQkT', 'iaso#2974'] for Zone de santé bu Bili Zone de Santé 21937 don't exist in input or SourceVersion` on the backend, not sure if it's related.

olethanh committed:
IA-1804 Rename InstancePopover to .tsx

olethanh committed:
IA-1804 convert InstancePopover to new style

beygorghor reviewed (changes requested):
Same here, adding .ts extension in `InstancesFilesListComponent`, `InstancesMapComponent` and `OrgUnitMapComponent` please

olethanh commented:
These files import InstancePop**Up** and not InstancePop**Over** because why have simple naming ? :rofl: 

Fixed the import, rebased on main, attributed jira ticket
